src/openrct2/world/Climate.h

```diff
@@ -38,19 +38,19 @@ enum WEATHER_EFFECT
     WEATHER_EFFECT_STORM,
 };
 
-enum RAIN_LEVEL
+enum class RainLevel_e
```

Comment:
We don't suffix/prefix our enums, please remove the suffix.

Reply:
How should handle the conflict with WeatherState::RainLevel?

Reply:
You can rename it to Level as the type already states its RainLevel